tests/tests_release.yml

```diff
@@ -7,14 +7,19 @@
   tags:
     - tests::slow
   tasks:
-    - name: Setup Candlepin
-      import_tasks: tasks/setup_candlepin.yml
+    # A local Candlepin deployment has no concept of "releases",
+    # so onlt setup the test data
```

Comment:
```suggestion
    # so only setup the test data
```

Reply:
fixed locally, will push once the CI completes (to check it works fine)

Reply:
ok - yeah, we won't need to do CI again after changing the comment